Configurations/descrip.mms.tmpl

```diff
@@ -441,6 +441,7 @@ test : tests
         @ ! {- output_off() if $disabled{tests}; "" -}
         DEFINE SRCTOP {- sourcedir() -}
         DEFINE BLDTOP {- builddir() -}
+        DEFINE FIPSKEY {- $config{FIPSKEY} -}
```

Comment:
With all the other build files, you create a make variable, which you refer to for the tests.  the same should be done here.

A make variable looks exactly the same in `descrip.mms`, i.e.:

```
FIPSKEY={- $config{FIPSKEY} -}
```

The DEFINE line would then look quite unsurprising:

```
        DEFINE FIPSKEY "$(FIPSKEY)"
```

Reply:
Okay.  I fixed SRCTOP/BLDTOP while I was there so that nobody else is led astray in the future :)